src/store/providers/tableColumns.ts

```diff
@@ -0,0 +1,28 @@
+import { makeVar } from '@apollo/client';
+
+const getTableColumns = (): Record<string, any> =>
+  JSON.parse(localStorage.getItem('tableColumns') || '{}') || {};
+export const configureColumns = (original: Array<any>, newOrder: Array<any>): Array<any> => {
+  const newDataOrder: Array<any> = [];
+  newOrder.forEach((newValue) => {
```

Comment:
I think that this forEach loop is absolutely correct. But personally, I would favor a more functional approach (reduce for instance)